test/xUnit/csharp/test_CompletionHelpers.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Management.Automation;
+using Xunit;
+
+namespace PSTests.Parallel
+{
+    public class CompletionHelpersTests
+    {
+        [Theory]
+        [InlineData("", "", "")]
+        [InlineData("\"", "", "\"")]
+        [InlineData("'", "", "'")]
+        [InlineData("\"word\"", "word", "\"")]
+        [InlineData("'word'", "word", "'")]
+        [InlineData("\"word", "word", "\"")]
+        [InlineData("'word", "word", "'")]
+        [InlineData("word\"", "word\"", "")]
+        [InlineData("word'", "word'", "")]
+        [InlineData("\"word's\"", "word's", "\"")]
```

Comment:
```suggestion
        [InlineData("'word\"s'", "word\"s", "'")]
```

Reply:
Added in f019f0f655a9ca5c7e6493eddc30a07960cd5fdc